### PR TITLE
Add Ultra TabSaver

### PR DESCRIPTION
**Pull request by morsamatias**
Add Ultra TabSaver to the list of apps.

## Project URL
https://github.com/morsamatias/UltraTabSaver

## Category
Extensions

## Description
open source Tabs Manager for Safari.
 
## Why it should be included to `Awesome macOS open source applications `
There isn't any other Tab Manager open source for safari.
## Checklist
- [x] Edit [applications.json](https://github.com/serhii-londar/open-source-mac-os-apps/blob/master/applications.json) instead of [README.md](https://github.com/serhii-londar/open-source-mac-os-apps/blob/master/README.md).
- [x] Only one project/change is in this pull request
- [x] Screenshots(s) added if any
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

**Comment by serhii-londar** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Found 327 link issues, a project collaborator will take care of these, thanks :)</td>
    </tr>
  </tbody>
</table>

#### Link issues by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
1239 | [429](https://httpstatuses.com/429) | https://github.com/remirobert/Github-contributions 
1255 | [429](https://httpstatuses.com/429) | https://github.com/desktop/desktop 
1271 | [429](https://httpstatuses.com/429) | https://github.com/ad/GithubListener 
1287 | [429](https://httpstatuses.com/429) | https://github.com/erik/github-notify 
1301 | [429](https://httpstatuses.com/429) | https://github.com/eonist/GitSync 
1317 | [429](https://httpstatuses.com/429) | https://github.com/git-up/GitUp 
1337 | [429](https://httpstatuses.com/429) | https://github.com/gitx/gitx 
1351 | [429](https://httpstatuses.com/429) | https://github.com/mtgto/GPM 
1365 | [429](https://httpstatuses.com/429) | https://github.com/glaurent/MessagesHistoryBrowser 
1379 | [429](https://httpstatuses.com/429) | https://github.com/jamieweavis/streaker 
1396 | [429](https://httpstatuses.com/429) | https://github.com/marcinreliga/TeamStatus-for-GitHub 
1412 | [429](https://httpstatuses.com/429) | https://github.com/ptsochantaris/trailer 
1429 | [429](https://httpstatuses.com/429) | https://github.com/Uncommon/Xit 
1445 | [429](https://httpstatuses.com/429) | https://github.com/MitMaro/git-interactive-rebase-tool 
1461 | [429](https://httpstatuses.com/429) | https://github.com/vineetchoudhary/AppBox-iOSAppsWirelessInstallation 
1479 | [429](https://httpstatuses.com/429) | https://github.com/kuyawa/AppIcons 
1495 | [429](https://httpstatuses.com/429) | https://github.com/everettjf/AppleTrace 
1509 | [429](https://httpstatuses.com/429) | https://github.com/arbel03/AppStoreReviewTimes 
1526 | [429](https://httpstatuses.com/429) | https://github.com/insidegui/AssetCatalogTinkerer 
1545 | [429](https://httpstatuses.com/429) | https://github.com/e7711bbear/Assets 
1559 | [429](https://httpstatuses.com/429) | https://github.com/attaswift/Attabench 
1577 | [429](https://httpstatuses.com/429) | https://github.com/angelvasa/AVXCAssets-Generator 
1591 | [429](https://httpstatuses.com/429) | https://github.com/JustinFincher/BoardForGitHub 
1609 | [429](https://httpstatuses.com/429) | https://github.com/br1sk/brisk 
1626 | [429](https://httpstatuses.com/429) | https://github.com/xing/calabash-launcher 
1646 | [429](https://httpstatuses.com/429) | https://github.com/waylybaye/XcodeCleaner 
1660 | [429](https://httpstatuses.com/429) | https://github.com/mmattozzi/cocoa-rest-client 
1680 | [429](https://httpstatuses.com/429) | https://github.com/KrisYu/FilterShop 
1696 | [429](https://httpstatuses.com/429) | https://github.com/onmyway133/IconGenerator 
1713 | [429](https://httpstatuses.com/429) | https://github.com/raphaelhanneken/iconizer 
1729 | [429](https://httpstatuses.com/429) | https://github.com/SAP/macos-icon-generator 
1743 | [429](https://httpstatuses.com/429) | https://github.com/johnno1962/InjectionIII 
1758 | [429](https://httpstatuses.com/429) | https://github.com/devcxm/iOS-Images-Extractor 
1775 | [429](https://httpstatuses.com/429) | https://github.com/wigl/iSimulator 
1789 | [429](https://httpstatuses.com/429) | https://github.com/KnuffApp/Knuff 
1805 | [429](https://httpstatuses.com/429) | https://github.com/yuhua-chen/LayerX 
1821 | [429](https://httpstatuses.com/429) | https://github.com/ralcr/Localizable.strings 
1835 | [429](https://httpstatuses.com/429) | https://github.com/e7711bbear/Localizations 
1849 | [429](https://httpstatuses.com/429) | https://github.com/igorkulman/iOSLocalizationEditor 
1863 | [429](https://httpstatuses.com/429) | https://github.com/nvzqz/Menubar-Colors 
1879 | [429](https://httpstatuses.com/429) | https://github.com/toxblh/MTMR 
1893 | [429](https://httpstatuses.com/429) | https://github.com/kizitonwose/PodsUpdater 
1910 | [429](https://httpstatuses.com/429) | https://github.com/shaojiankui/ProfilesManager 
1926 | [429](https://httpstatuses.com/429) | https://github.com/onmyway133/PushNotifications 
1950 | [429](https://httpstatuses.com/429) | https://github.com/InjoyDeng/ResignTool 
1966 | [429](https://httpstatuses.com/429) | https://github.com/onurgenes/Resizr 
1980 | [429](https://httpstatuses.com/429) | https://github.com/shaojiankui/SmartPush 
1996 | [429](https://httpstatuses.com/429) | https://github.com/touchbar/Touch-Bar-Preview 
2010 | [429](https://httpstatuses.com/429) | https://github.com/sindresorhus/touch-bar-simulator 
2024 | [429](https://httpstatuses.com/429) | https://github.com/iseebi/TransporterPad 
2038 | [429](https://httpstatuses.com/429) | https://github.com/insidegui/WWDC 
2052 | [429](https://httpstatuses.com/429) | https://github.com/ssamadgh/WWDCsrt 
2072 | [429](https://httpstatuses.com/429) | https://github.com/novemberfiveco/xib2Storyboard 
2086 | [429](https://httpstatuses.com/429) | https://github.com/zadr/j2s 
2100 | [429](https://httpstatuses.com/429) | https://github.com/AppCraft-LLC/json-mapper 
2114 | [429](https://httpstatuses.com/429) | https://github.com/Ahmed-Ali/JSONExport 
2130 | [429](https://httpstatuses.com/429) | https://github.com/RafalWilinski/s3-uploader 
2146 | [429](https://httpstatuses.com/429) | https://github.com/TheNewNormal/corectl.app 
2162 | [429](https://httpstatuses.com/429) | https://github.com/TheNewNormal/coreos-osx 
2178 | [429](https://httpstatuses.com/429) | https://github.com/httptoolkit/httptoolkit-desktop 
2196 | [429](https://httpstatuses.com/429) | https://github.com/Kong/insomnia 
2212 | [429](https://httpstatuses.com/429) | https://github.com/trulyronak/itunesconnect 
2226 | [429](https://httpstatuses.com/429) | https://github.com/DanSanche/KubeMonitor 
2242 | [429](https://httpstatuses.com/429) | https://github.com/nsriram/KubeSwitch 
2258 | [429](https://httpstatuses.com/429) | https://github.com/RoyalIcing/Lantern 
2272 | [429](https://httpstatuses.com/429) | https://github.com/plan44/localSites 
2286 | [429](https://httpstatuses.com/429) | https://github.com/andypotts/mockup-generator 
2303 | [429](https://httpstatuses.com/429) | https://github.com/720kb/ndm 
2319 | [429](https://httpstatuses.com/429) | https://github.com/vsaravind007/nodeScratchpad 
2333 | [429](https://httpstatuses.com/429) | https://github.com/zeit/now-desktop 
2349 | [429](https://httpstatuses.com/429) | https://github.com/inket/stts 
2365 | [429](https://httpstatuses.com/429) | https://github.com/Yasumoto/ChefInspector 
2381 | [429](https://httpstatuses.com/429) | https://github.com/dcsch/macho-browser 
2395 | [429](https://httpstatuses.com/429) | https://github.com/ant4g0nist/vegvisir 
2418 | [429](https://httpstatuses.com/429) | https://github.com/yep/app-downloader 
2432 | [429](https://httpstatuses.com/429) | https://github.com/Kevin-De-Koninck/Get-It 
2446 | [429](https://httpstatuses.com/429) | https://github.com/agalwood/Motrix 
2448 | [429](https://httpstatuses.com/429) | https://github.com/agalwood/Motrix/blob/master/build/256x256.png 
2465 | [429](https://httpstatuses.com/429) | https://github.com/Pjirlip/Pillager 
2483 | [429](https://httpstatuses.com/429) | https://github.com/FaisalUmair/udemy-downloader-gui 
2497 | [429](https://httpstatuses.com/429) | https://github.com/DenBeke/YouTube-Downloader-for-macOS 
2513 | [429](https://httpstatuses.com/429) | https://github.com/jakob/TableTool 
2527 | [429](https://httpstatuses.com/429) | https://github.com/wellsjo/JSON-Splora 
2544 | [429](https://httpstatuses.com/429) | https://github.com/gingko/client 
2560 | [429](https://httpstatuses.com/429) | https://github.com/MacDownApp/macdown 
2574 | [429](https://httpstatuses.com/429) | https://github.com/marktext/marktext/ 
2588 | [429](https://httpstatuses.com/429) | https://github.com/lukakerr/Pine 
2606 | [429](https://httpstatuses.com/429) | https://github.com/qilin-editor/qilin-app 
2624 | [429](https://httpstatuses.com/429) | https://github.com/coteditor/CotEditor 
2640 | [429](https://httpstatuses.com/429) | https://github.com/macvim-dev/macvim 
2654 | [429](https://httpstatuses.com/429) | https://github.com/brunophilipe/noto 
2671 | [429](https://httpstatuses.com/429) | https://github.com/textmate/textmate 
2687 | [429](https://httpstatuses.com/429) | https://github.com/qvacua/vimr 
2701 | [429](https://httpstatuses.com/429) | https://github.com/Capevace/BetterPiP 
2717 | [429](https://httpstatuses.com/429) | https://github.com/arnoappenzeller/PiPifier 
2735 | [429](https://httpstatuses.com/429) | https://github.com/bfmatei/PiPTool 
2749 | [429](https://httpstatuses.com/429) | https://github.com/bow-swift/nef-plugin 
2766 | [429](https://httpstatuses.com/429) | https://github.com/Clipy/Clipy 
2783 | [429](https://httpstatuses.com/429) | https://github.com/hluk/CopyQ 
2799 | [429](https://httpstatuses.com/429) | https://github.com/onmyway133/FinderGo 
2813 | [429](https://httpstatuses.com/429) | https://github.com/Mortennn/FiScript 
2830 | [429](https://httpstatuses.com/429) | https://github.com/sozercan/OpenInCode 
2846 | [429](https://httpstatuses.com/429) | https://github.com/wesnoth/wesnoth 
2860 | [429](https://httpstatuses.com/429) | https://github.com/alunbestor/Boxer 
2875 | [429](https://httpstatuses.com/429) | https://github.com/dolphin-emu/dolphin 
2889 | [429](https://httpstatuses.com/429) | https://github.com/OpenEmu/OpenEmu 
2905 | [429](https://httpstatuses.com/429) | https://github.com/OpenRCT2/OpenRCT2 
2921 | [429](https://httpstatuses.com/429) | https://github.com/AaronRandall/Screentendo 
2937 | [429](https://httpstatuses.com/429) | https://github.com/daylen/stockfish-mac 
2952 | [429](https://httpstatuses.com/429) | https://github.com/aseprite/aseprite 
2967 | [429](https://httpstatuses.com/429) | https://github.com/gragrance/CaptuocrToy 
2983 | [429](https://httpstatuses.com/429) | https://github.com/onmyway133/GifCapture 
3003 | [429](https://httpstatuses.com/429) | https://github.com/lettier/gifcurry 
3020 | [429](https://httpstatuses.com/429) | https://github.com/sindresorhus/Gifski 
3039 | [429](https://httpstatuses.com/429) | https://github.com/CleanCocoa/InfiniteCanvas 
3055 | [429](https://httpstatuses.com/429) | https://github.com/dsward2/macSVG 
3071 | [429](https://httpstatuses.com/429) | https://github.com/BafS/Material-Colors-native 
3087 | [429](https://httpstatuses.com/429) | https://github.com/pencil2d/pencil 
3101 | [429](https://httpstatuses.com/429) | https://github.com/duyquoc/ScreenToLayers 
3118 | [429](https://httpstatuses.com/429) | https://github.com/atom/atom 
3132 | [429](https://httpstatuses.com/429) | https://github.com/onivim/oni 
3147 | [429](https://httpstatuses.com/429) | https://github.com/Microsoft/vscode 
3163 | [429](https://httpstatuses.com/429) | https://github.com/pkulchenko/ZeroBraneStudio 
3179 | [429](https://httpstatuses.com/429) | https://github.com/shgodoroja/APNGb 
3193 | [429](https://httpstatuses.com/429) | https://github.com/szTheory/exifcleaner 
3211 | [429](https://httpstatuses.com/429) | https://github.com/chrissimpkins/Crunch 
3225 | [429](https://httpstatuses.com/429) | https://github.com/wddwycc/Freehand 
3241 | [429](https://httpstatuses.com/429) | https://github.com/GNOME/gimp 
3255 | [429](https://httpstatuses.com/429) | https://github.com/liamrosenfeld/Iconology 
3269 | [429](https://httpstatuses.com/429) | https://github.com/kornelski/ImageAlpha 
3286 | [429](https://httpstatuses.com/429) | https://github.com/meowtec/Imagine 
3300 | [429](https://httpstatuses.com/429) | https://github.com/bluegill/katana 
3315 | [429](https://httpstatuses.com/429) | https://github.com/gergelysanta/photominer 
3336 | [429](https://httpstatuses.com/429) | https://github.com/crilleengvall/Screenbar 
3352 | [429](https://httpstatuses.com/429) | https://github.com/1000ch/WebPonize 
3367 | [429](https://httpstatuses.com/429) | https://github.com/msvisser/AnnePro-mac 
3381 | [429](https://httpstatuses.com/429) | https://github.com/Pyroh/Fluor 
3395 | [429](https://httpstatuses.com/429) | https://github.com/yqrashawn/GokuRakuJoudo 
3409 | [429](https://httpstatuses.com/429) | https://github.com/tekezo/Karabiner 
3424 | [429](https://httpstatuses.com/429) | https://github.com/pqrs-org/Karabiner-Elements 
3439 | [429](https://httpstatuses.com/429) | https://github.com/utatti/kawa 
3456 | [429](https://httpstatuses.com/429) | https://github.com/gbammc/Thor 
3473 | [429](https://httpstatuses.com/429) | https://github.com/aahung/Unshaky 
3477 | [429](https://httpstatuses.com/429) | https://github.com/aahung/Unshaky/blob/master/README/menubar.png 
3478 | [429](https://httpstatuses.com/429) | https://github.com/aahung/Unshaky/blob/master/README/how-unshaky-works.png 
3490 | [429](https://httpstatuses.com/429) | https://github.com/amitmerchant1990/correo 
3504 | [429](https://httpstatuses.com/429) | https://github.com/dinhviethoa/dejalu 
3521 | [429](https://httpstatuses.com/429) | https://github.com/Foundry376/Mailspring 
3535 | [429](https://httpstatuses.com/429) | https://github.com/ramboxapp/community-edition 
3551 | [429](https://httpstatuses.com/429) | https://github.com/invesalius/invesalius3/ 
3569 | [429](https://httpstatuses.com/429) | https://github.com/alvesjtiago/airpass 
3583 | [429](https://httpstatuses.com/429) | https://github.com/tonsky/AnyBar 
3597 | [429](https://httpstatuses.com/429) | https://github.com/matryer/bitbar 
3611 | [429](https://httpstatuses.com/429) | https://github.com/josh-/CloudyTabs 
3627 | [429](https://httpstatuses.com/429) | https://github.com/inderdhir/DatWeatherDoe 
3641 | [429](https://httpstatuses.com/429) | https://github.com/Kwpolska/DisplayMenu 
3655 | [429](https://httpstatuses.com/429) | https://github.com/CodySchrank/gSwitch 
3669 | [429](https://httpstatuses.com/429) | https://github.com/iglance/iGlance 
3683 | [429](https://httpstatuses.com/429) | https://github.com/turkenh/KubeContext 
3700 | [429](https://httpstatuses.com/429) | https://github.com/lucasbento/menubar-brightness 
3714 | [429](https://httpstatuses.com/429) | https://github.com/yujitach/MenuMeters 
3728 | [429](https://httpstatuses.com/429) | https://github.com/isaiasmatewos/night-shift-control 
3744 | [429](https://httpstatuses.com/429) | https://github.com/nikhilsh/PSIBar 
3758 | [429](https://httpstatuses.com/429) | https://github.com/alexrosenfeld10/Quickeys 
3774 | [429](https://httpstatuses.com/429) | https://github.com/archagon/sensible-side-buttons 
3789 | [429](https://httpstatuses.com/429) | https://github.com/thompsonate/Shifty 
3805 | [429](https://httpstatuses.com/429) | https://github.com/AnaghSharma/Carol 
3819 | [429](https://httpstatuses.com/429) | https://github.com/cemolcay/ChordDetector 
3833 | [429](https://httpstatuses.com/429) | https://github.com/imanel/deezplayer 
3847 | [429](https://httpstatuses.com/429) | https://github.com/Zac-Garby/iTunes-Graphs 
3864 | [429](https://httpstatuses.com/429) | https://github.com/mamal72/lyricsify-mac 
3881 | [429](https://httpstatuses.com/429) | https://github.com/salomvary/soundcleod 
3897 | [429](https://httpstatuses.com/429) | https://github.com/ersel/spotify-cli-mac 
3913 | [429](https://httpstatuses.com/429) | https://github.com/steve228uk/YouTube-Music 
3932 | [429](https://httpstatuses.com/429) | https://github.com/ngquerol/Diurna 
3948 | [429](https://httpstatuses.com/429) | https://github.com/jingweno/hacker-menu 
3964 | [429](https://httpstatuses.com/429) | https://github.com/Ranchero-Software/NetNewsWire 
3978 | [429](https://httpstatuses.com/429) | https://github.com/ViennaRSS/vienna-rss 
3992 | [429](https://httpstatuses.com/429) | https://github.com/BoostIO/Boostnote 
4008 | [429](https://httpstatuses.com/429) | https://github.com/Kilian/fromscratch 
4023 | [429](https://httpstatuses.com/429) | https://github.com/glushchenko/fsnotes 
4040 | [429](https://httpstatuses.com/429) | https://github.com/laurent22/joplin 
4054 | [429](https://httpstatuses.com/429) | https://github.com/tuxu/nbviewer-app 
4068 | [429](https://httpstatuses.com/429) | https://github.com/jmcfarlane/notable 
4090 | [429](https://httpstatuses.com/429) | https://github.com/SauvageP/Notes 
4104 | [429](https://httpstatuses.com/429) | https://github.com/insidegui/NoteTaker 
4120 | [429](https://httpstatuses.com/429) | https://github.com/Automattic/simplenote-macos 
4136 | [429](https://httpstatuses.com/429) | https://github.com/standardnotes/web 
4153 | [429](https://httpstatuses.com/429) | https://github.com/buddax2/tmpNote 
4169 | [429](https://httpstatuses.com/429) | https://github.com/klaussinani/tusk 
4186 | [429](https://httpstatuses.com/429) | https://github.com/pbek/QOwnNotes 
4202 | [429](https://httpstatuses.com/429) | https://github.com/brunophilipe/Cakebrew 
4218 | [429](https://httpstatuses.com/429) | https://github.com/dteoh/devdocs-macos 
4234 | [429](https://httpstatuses.com/429) | https://github.com/2ndalpha/gasmask 
4248 | [429](https://httpstatuses.com/429) | https://github.com/specialunderwear/Hosts.prefpane 
4262 | [429](https://httpstatuses.com/429) | https://github.com/ImageOptim/ImageOptim 
4276 | [429](https://httpstatuses.com/429) | https://github.com/insidegui/KeyframesPlayer 
4292 | [429](https://httpstatuses.com/429) | https://github.com/hackjutsu/Lepton 
4306 | [429](https://httpstatuses.com/429) | https://github.com/Bunn/macGist 
4320 | [429](https://httpstatuses.com/429) | https://github.com/sveinbjornt/Platypus 
4334 | [429](https://httpstatuses.com/429) | https://github.com/goktugyil/QorumLogs 
4348 | [429](https://httpstatuses.com/429) | https://github.com/jhen0409/react-native-debugger 
4364 | [429](https://httpstatuses.com/429) | https://github.com/infinitered/reactotron 
4378 | [429](https://httpstatuses.com/429) | https://github.com/woofwoofinc/rktmachine 
4395 | [429](https://httpstatuses.com/429) | https://github.com/gosu/ruby-app 
4409 | [429](https://httpstatuses.com/429) | https://github.com/fitztrev/shuttle 
4425 | [429](https://httpstatuses.com/429) | https://github.com/SwiftyBeaver/SwiftyBeaver 
4439 | [429](https://httpstatuses.com/429) | https://github.com/swiftyapp/swifty 
4456 | [429](https://httpstatuses.com/429) | https://github.com/syncthing/syncthing-macos 
4472 | [429](https://httpstatuses.com/429) | https://github.com/jeffhodnett/Unused 
4486 | [429](https://httpstatuses.com/429) | https://github.com/lanayotech/vagrant-manager 
4502 | [429](https://httpstatuses.com/429) | https://github.com/z-------------/CPod 
4518 | [429](https://httpstatuses.com/429) | https://github.com/muammar/mkchromecast 
4532 | [429](https://httpstatuses.com/429) | https://github.com/insidegui/PodcastMenu 
4546 | [429](https://httpstatuses.com/429) | https://github.com/phranck/podlive-macos 
4558 | [429](https://httpstatuses.com/429) | https://github.com/dyerc/Doughnut 
4578 | [429](https://httpstatuses.com/429) | https://github.com/klaussinani/ao 
4602 | [429](https://httpstatuses.com/429) | https://github.com/KELiON/cerebro 
4616 | [429](https://httpstatuses.com/429) | https://github.com/naotaka/ClipMenu 
4632 | [429](https://httpstatuses.com/429) | https://github.com/n0shake/Clocker 
4646 | [429](https://httpstatuses.com/429) | https://github.com/dustinrue/ControlPlane 
4660 | [429](https://httpstatuses.com/429) | https://github.com/TermiT/flycut 
4674 | [429](https://httpstatuses.com/429) | https://github.com/federico-terzi/espanso 
4690 | [429](https://httpstatuses.com/429) | https://github.com/Clipy/KeyHolder 
4704 | [429](https://httpstatuses.com/429) | https://github.com/kiwix/apple 
4718 | [429](https://httpstatuses.com/429) | https://github.com/fespinoza/LinkedIdeas 
4734 | [429](https://httpstatuses.com/429) | https://github.com/p0deje/Maccy 
4750 | [429](https://httpstatuses.com/429) | https://github.com/shubhambatra3019/macOrganizer 
4764 | [429](https://httpstatuses.com/429) | https://github.com/hql287/Manta 
4778 | [429](https://httpstatuses.com/429) | https://github.com/thomasbrueggemann/paperless-desktop 
4795 | [429](https://httpstatuses.com/429) | https://github.com/PDF-Archiver/PDF-Archiver 
4811 | [429](https://httpstatuses.com/429) | https://github.com/quicksilver/Quicksilver 
4825 | [429](https://httpstatuses.com/429) | https://github.com/quickwords/quickwords 
4842 | [429](https://httpstatuses.com/429) | https://github.com/SelfControlApp/selfcontrol 
4856 | [429](https://httpstatuses.com/429) | https://github.com/Onix-Systems/osx-status-bar-todo 
4872 | [429](https://httpstatuses.com/429) | https://github.com/hovancik/stretchly 
4886 | [429](https://httpstatuses.com/429) | https://github.com/joaomoreno/thyme 
4900 | [429](https://httpstatuses.com/429) | https://github.com/michaelvillar/timer-app 
4930 | [429](https://httpstatuses.com/429) | https://github.com/jlong/TrelloApp 
4944 | [429](https://httpstatuses.com/429) | https://github.com/TailorDev/Watson 
4961 | [429](https://httpstatuses.com/429) | https://github.com/1000ch/whale 
4977 | [429](https://httpstatuses.com/429) | https://github.com/sendyhalim/Yomu 
4991 | [429](https://httpstatuses.com/429) | https://github.com/JohnCoates/Aerial 
5012 | [429](https://httpstatuses.com/429) | https://github.com/ved62/Image-As-Wallpaper 
5026 | [429](https://httpstatuses.com/429) | https://github.com/leonspok/Irvue-Screensaver 
5044 | [429](https://httpstatuses.com/429) | https://github.com/docterd/MusaicFM 
5060 | [429](https://httpstatuses.com/429) | https://github.com/winterbe/github-matrix-screensaver 
5077 | [429](https://httpstatuses.com/429) | https://github.com/objective-see/LuLu 
5091 | [429](https://httpstatuses.com/429) | https://github.com/alichtman/stronghold 
5122 | [429](https://httpstatuses.com/429) | https://github.com/mileswd/mac2imgur 
5136 | [429](https://httpstatuses.com/429) | https://github.com/nitroshare/nitroshare-desktop 
5150 | [429](https://httpstatuses.com/429) | https://github.com/qbittorrent/qBittorrent 
5164 | [429](https://httpstatuses.com/429) | https://github.com/timonus/Rhea 
5178 | [429](https://httpstatuses.com/429) | https://github.com/transmission/transmission 
5193 | [429](https://httpstatuses.com/429) | https://github.com/Tribler/tribler 
5207 | [429](https://httpstatuses.com/429) | https://github.com/vitorgalvao/appear.in 
5221 | [429](https://httpstatuses.com/429) | https://github.com/sindresorhus/caprine#features 
5236 | [429](https://httpstatuses.com/429) | https://github.com/danielbuechele/goofy 
5250 | [429](https://httpstatuses.com/429) | https://github.com/Swiftodon/Leviathan 
5264 | [429](https://httpstatuses.com/429) | https://github.com/rsms/fb-mac-messenger 
5280 | [429](https://httpstatuses.com/429) | https://github.com/producthunt/producthunt-osx 
5294 | [429](https://httpstatuses.com/429) | https://github.com/1000ch/quail 
5310 | [429](https://httpstatuses.com/429) | https://github.com/terkelg/ramme 
5325 | [429](https://httpstatuses.com/429) | https://github.com/KeliCheng/Simpo 
5342 | [429](https://httpstatuses.com/429) | https://github.com/michealparks/galeri 
5356 | [429](https://httpstatuses.com/429) | https://github.com/obsproject/obs-studio 
5370 | [429](https://httpstatuses.com/429) | https://github.com/raphaelhanneken/apple-juice 
5388 | [429](https://httpstatuses.com/429) | https://github.com/AppPolice/AppPolice 
5408 | [429](https://httpstatuses.com/429) | https://github.com/Kevin-De-Koninck/Clean-Me 
5424 | [429](https://httpstatuses.com/429) | https://github.com/macmade/Diagnostics 
5440 | [429](https://httpstatuses.com/429) | https://github.com/Eun/DisableMonitor 
5454 | [429](https://httpstatuses.com/429) | https://github.com/etresoft/EtreCheck 
5468 | [429](https://httpstatuses.com/429) | https://github.com/DanielStormApps/Fanny 
5484 | [429](https://httpstatuses.com/429) | https://github.com/jwise/HoRNDIS 
5498 | [429](https://httpstatuses.com/429) | https://github.com/brianmichel/Juice 
5515 | [429](https://httpstatuses.com/429) | https://github.com/newmarcel/KeepingYouAwake 
5531 | [429](https://httpstatuses.com/429) | https://github.com/mangerlahn/Latest 
5547 | [429](https://httpstatuses.com/429) | https://github.com/BonzaiThePenguin/Loading 
5563 | [429](https://httpstatuses.com/429) | https://github.com/KrauseFx/overkill-for-mac 
5577 | [429](https://httpstatuses.com/429) | https://github.com/ProfileCreator/ProfileCreator 
5593 | [429](https://httpstatuses.com/429) | https://github.com/sveinbjornt/Sloth 
5611 | [429](https://httpstatuses.com/429) | https://github.com/rugarciap/Turbo-Boost-Switcher 
5625 | [429](https://httpstatuses.com/429) | https://github.com/DeromirNeves/VerticalBar 
5641 | [429](https://httpstatuses.com/429) | https://github.com/alacritty/alacritty 
5657 | [429](https://httpstatuses.com/429) | https://github.com/ishuah/bifrost 
5671 | [429](https://httpstatuses.com/429) | https://github.com/macmade/Console 
5687 | [429](https://httpstatuses.com/429) | https://github.com/zeit/hyper 
5704 | [429](https://httpstatuses.com/429) | https://github.com/gnachman/iTerm2 
5718 | [429](https://httpstatuses.com/429) | https://github.com/kovidgoyal/kitty 
5733 | [429](https://httpstatuses.com/429) | https://github.com/es-kumagai/OpenTerminal 
5747 | [429](https://httpstatuses.com/429) | https://github.com/railsware/upterm 
7210 | [429](https://httpstatuses.com/429) | https://github.com/pigigaldi/Pock 
7222 | [429](https://httpstatuses.com/429) | https://github.com/macmade/Notarize 
7238 | [429](https://httpstatuses.com/429) | https://github.com/vpeschenkov/Predator 
7255 | [429](https://httpstatuses.com/429) | https://github.com/amiantos/lifesaver 
7261 | [429](https://httpstatuses.com/429) | https://github.com/amiantos/lifesaver/blob/master/Design/tvos/tvos-ss-1.png 
7262 | [429](https://httpstatuses.com/429) | https://github.com/amiantos/lifesaver/blob/master/Design/tvos/tvos-ss-2.png 
7273 | [429](https://httpstatuses.com/429) | https://github.com/LumingYin/Caption 
7289 | [429](https://httpstatuses.com/429) | https://github.com/LumingYin/SpeedReader 
7294 | [429](https://httpstatuses.com/429) | https://github.com/LumingYin/SpeedReader/blob/master/speedreader_dark.gif 
7309 | [429](https://httpstatuses.com/429) | https://github.com/rxhanson/Rectangle 
7325 | [429](https://httpstatuses.com/429) | https://github.com/Mortennn/Dozer 
7341 | [429](https://httpstatuses.com/429) | https://github.com/audacity/audacity 
7363 | [429](https://httpstatuses.com/429) | https://github.com/doches/lilypond-ui 
7373 | [429](https://httpstatuses.com/429) | https://github.com/fphilipe/PageExtender.app 
7390 | [429](https://httpstatuses.com/429) | https://github.com/yonilevy/automute 
7407 | [429](https://httpstatuses.com/429) | https://github.com/ExistentialAudio/BlackHole 
7435 | [429](https://httpstatuses.com/429) | https://github.com/jhspetersson/fselect 
7449 | [429](https://httpstatuses.com/429) | https://github.com/livecode/livecode 
7463 | [429](https://httpstatuses.com/429) | https://github.com/ivoronin/TomatoBar 
7477 | [429](https://httpstatuses.com/429) | https://github.com/ivoronin/ArchiveMounter 
7491 | [429](https://httpstatuses.com/429) | https://github.com/nikolaeu/Numi 
7505 | [429](https://httpstatuses.com/429) | https://github.com/powershell/powershell 
7519 | [429](https://httpstatuses.com/429) | https://github.com/cryptomator/cryptomator 
7539 | [429](https://httpstatuses.com/429) | https://github.com/johannesjo/super-productivity 
7559 | [429](https://httpstatuses.com/429) | https://github.com/varol/Calculeta 
7574 | [429](https://httpstatuses.com/429) | https://github.com/khrykin/StrategrDesktop 
7592 | [429](https://httpstatuses.com/429) | https://github.com/jeromelebel/MongoHub-Mac 
7608 | [429](https://httpstatuses.com/429) | https://github.com/AlexPerathoner/Sessions 
7625 | [429](https://httpstatuses.com/429) | https://github.com/sindresorhus/Plash 
7642 | [429](https://httpstatuses.com/429) | https://github.com/kamranahmedse/pennywise 
7661 | [429](https://httpstatuses.com/429) | https://github.com/dnote/dnote 
7680 | [429](https://httpstatuses.com/429) | https://github.com/DaFuqtor/MiddleClick-Catalina 
7684 | [429](https://httpstatuses.com/429) | https://github.com/DaFuqtor/MiddleClick-Catalina/blob/master/demo.png 
7692 | [429](https://httpstatuses.com/429) | https://github.com/jet8a/pomodoro-cycle-app 
7693 | [429](https://httpstatuses.com/429) | https://github.com/jet8a/pomodoro-cycle-app/releases 
7713 | [429](https://httpstatuses.com/429) | https://github.com/emqx/MQTTX 


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by morsamatias** [comment]
@serhii-londar  ???

**Comment by morsamatias** [comment]
@serhii-londar

**Comment by serhii-londar** [comment]
@morsamatias Thank you! 👍 Sorry for the delay.

File: applications.json [hunk 1]
```diff
@@ -19,6 +19,25 @@
...
+           "repo_url": "https://github.com/morsamatias/UltraTabSaver",
+           "title": "Ultra TabSaver",
+           "icon_url": "https://raw.githubusercontent.com/morsamatias/UltraTabSaver/master/Ultra%20TabSaver/Assets.xcassets/AppIcon.appiconset/unnamed-1.png",
+           "screenshots": [
+              "https://raw.githubusercontent.com/morsamatias/UltraTabSaver/master/Screenshoot-1.png",
+              "https://raw.githubusercontent.com/morsamatias/UltraTabSaver/master/Screenshoot-2.png"
+           ],
+           "official_site": "",
+           "languages": [
+               "Swift"
```
Comment (serhii-londar) [inline applications.json:38]: @morsamatias Please use lowercased string for language.

Comment (morsamatias) [reply applications.json]: I already change it 😃

**Comment by serhii-londar** [comment]
@morsamatias Thank you! 👍

**Pull request merged**